Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:12-alpine
+
+RUN mkdir /app
+WORKDIR /app
+ADD . .
+RUN yarn install
+
+FROM nginx:1.15-alpine
+
+RUN apk add --no-cache --update bash curl && \
+    rm -rf /var/cache/apk/*
+
+ARG app_name=unleash-frontend
+ARG build_version="1.0.2-edge"
+
+EXPOSE 80
+COPY  --from=builder /app/build /usr/share/nginx/html
```

Comment:
Does this automatically reference the node build even if it's not defined as `builder` above?

Reply:
it successfully builds the application and I use this pattern from other FE services :D

Reply:
Good to know! :smile: